openslides_backend/action/actions/user/set_present.py

```diff
@@ -71,6 +71,11 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
             self.user_id,
             Permissions.User.CAN_MANAGE,
             instance["meeting_id"],
+        ) or has_perm(
```

Comment:
You can remove the permission check above since the permissions are nested

Reply:
done